rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -152,20 +207,23 @@ def additional_update(self, *args, **kwargs) -> List[Mapping[str, Any]]:
         By default this is a pass through that calls `RLTrainer.additional_update`
 
         Args:
-            *args: Arguments to pass to each RLTrainer.
+            reduce_fn: See `update()` documentation for more details.
```

Comment:
ah lets instead have the same description in both places

Reply:
users wont always be reading src code

Reply:
It's difficult to maintain and can quickly result in regression. This is more explicit so that you only have to update one place if things change.